doc/setup.md

```diff
@@ -94,7 +103,44 @@ These instructions assume that you have [Homebrew](http://brew.sh/) installed.
    conda activate skyportal_env
    conda install ligo.skymap
    ```
+<a name="configure-shell-mac"></a>
+### Configuring Shell Environment for Development
+
+When developing with SkyPortal on mac, you may  also need to configure your shell enviroment. Here is how you can do that by editing your `.zshrc` file.
 
+#### Open your `.zshrc` file in a text editor:
+
+```
+nano ~/.zshrc
+```
+#### Set enviroment variables to their installation paths 
+After installing the libraries with Homebrew, you'll need to set environment variables to their installation paths. Typically, Homebrew provides these paths upon successful installation. Replace the placeholder text `<path-to-library>` with the actual path that Homebrew provides upon sucessful installation.
```

Comment:
Nice! One super useful command you can mention here is: `brew info <name_of_package>` that tells you where something is installed even after you installed it. Useful to figure out the installation paths of packages you alteady have